models/training-tuning-scripts/phishing-models/phish-bert-training.ipynb

```diff
@@ -419,7 +419,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.13"
+   "version": "3.6.9 (default, Jan 26 2021, 15:33:00) \n[GCC 8.4.0]"
```

Comment:
Ugh I hate that Jupyter saves this stuff, and this is probably nit-picky, but Morpheus requires Python 3.8

Reply:
I think that's just the Python version that was being used when the notebook was first created. It's not saying that it requires 3.6.9 to run. The kernel you specify for the notebook will determine the version of Python that it uses.